easybuild/easyconfigs/f/FLANN/FLANN-1.9.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,45 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'CMakeMake'
+
+name = 'FLANN'
+version = '1.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/mariusmuja/flann/'
+description = "FLANN is a library for performing fast approximate nearest neighbor searches in high dimensional spaces."
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = [
+    'b23b5f4e71139faa3bcb39e6bbcc76967fbaf308c4ee9d4f5bfbeceaa76cc5d3',  # flann-1.9.1.tar.gz
+    '479068a9f1c745780a7fbf55e6ba02011d5a6ca0a917ce0d4e11367c14a8cf78',  # cmake-flann-1.9.1.patch
+]
+
+builddependencies = [('CMake', '3.16.4')]
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-3.8.2'),
+]
+
+separate_build_dir = True
+
+patches = ['%(name)s-%(version)s_fix-cmake.patch']
```

Comment:
```suggestion
```

Reply:
Changed to `patches = ['%(name)s-fix-cmake.patch']` as above